File: algoritmos/Algoritmo 2/Algoritmo2/src/main/java/com/fabio/algoritmo2/Andorinha.java
package com.fabio.algoritmo2;

/**
 *
 * @author fabio
 */

public class Andorinha extends Ave implements AveVoo{
    
    public Andorinha(double peso, int n_patas){
        super(peso, n_patas);
    }
    
    @Override
    public void voar(){
        System.out.println("estou voando.\n");
    }
}

File: algoritmos/Exercício 2/Exercicio2_UEPG/Classes/andorinha.js
/*Andorinhas e patos implementam uma interface para o vôo.*/
class Andorinha extends Ave{
    constructor(peso, patas){
        super(peso, patas);
    }

    voar() {
        let mensagem = "Sou uma andorinha e estou voando";
        alert(mensagem);
        return mensagem;
    }    
}